src/features/company/redux/middleware.ts

```diff
@@ -0,0 +1,21 @@
+import { BOOTSTRAP } from "../../../store/constants";
+import { updateCompanyOverviewAction, CompanyOverview } from "./actions";
+import { SocketService } from "../../../services";
+import { Middleware } from "redux";
+import { AppState } from "../../../store";
+
+type Dependencies = {
+  socketService: SocketService;
```

Comment:
I see that [all middlewares get the same dependencies](https://github.com/steinalex/stock-tracker/blob/development/src/store/store.js) so it might be nice to move this type to `store` and share it?

Reply:
Kyle mentioned that it might be better to specify the dependencies in each middleware file. Thoughts @Sharakai?